libcudacxx/include/cuda/std/detail/libcxx/include/chrono

```diff
@@ -3548,7 +3548,10 @@ _LIBCUDACXX_HIDE_FROM_ABI constexpr hours make24(const hours& __h, bool __is_pm)
 _CCCL_DIAG_PUSH
 _CCCL_DIAG_SUPPRESS_GCC("-Wliteral-suffix")
 _CCCL_DIAG_SUPPRESS_CLANG("-Wuser-defined-literals")
+_CCCL_DIAG_SUPPRESS_NVHPC(lit_suffix_no_underscore)
 _CCCL_DIAG_SUPPRESS_MSVC(4455)
+_CCCL_NV_DIAG_SUPPRESS(2506)
```

Comment:
Please add a comment on the warning we are suppressing, the numbers are always a bit cryptic